queryserver/src/main/java/org/apache/phoenix/util/HostAndPort.java

```diff
@@ -0,0 +1,284 @@
+/*
+ * Copyright (C) 2011 The Guava Authors
```

Comment:
We should ensure that a top-level NOTICE file propagates this copyright notice, as well as the containing jar.

the ASF parent pom already has convention to append entries to the NOTICE file. One example https://github.com/apache/hbase/blob/master/hbase-thrift/src/main/appended-resources/META-INF/NOTICE. The "src/main/appended-resources" is the magic set up by asf.pom.